app/controllers/api/v1x0/mixins/rbac_mixin.rb

```diff
@@ -28,11 +28,7 @@ def permission_check(verb, klass = controller_name.classify.constantize)
         end
 
         def permission_array_check(permissions)
-          if !permissions.kind_of?(Array)
-            invalid_parameter('Permission should be an array')
-          elsif permissions.blank? || permissions.any?(&:blank?)
-            invalid_parameter('Permissions should not be empty')
-          end
+          invalid_parameter('Permission should be an array') unless permissions.kind_of?(Array)
```

Comment:
This is now caught by https://github.com/ManageIQ/catalog-api/blob/master/app/controllers/api/v1x0/portfolios_controller.rb#L10, since we're doing a `params.require(:permissions)`, if permissions is an empty array, we get back an error.